types/node/index.d.ts

```diff
@@ -4708,6 +4709,7 @@ declare module "tls" {
         secureContext?: Object;
         session?: Buffer;
         minDHSize?: number;
+        lookup?: (hostname: string, options: dns.LookupOneOptions | dns.LookupOptions, callback: (err: NodeJS.ErrnoException, address: string, family?: number) => void) => void;
```

Comment:
- Is it really needed to include `dns.LookupOptions` here? I think `dns.LookupOneOptions? should be enough.
- I think `err` should be `NodeJS.ErrnoException | null` (should be changed at several places but this is not the point of this PR).
- Why is `family` optional here?

Reply:
Thanks for your quick review @Flarna!

> Is it really needed to include dns.LookupOptions here? I think `dns.LookupOneOptions? should be enough.

Actually, [reading the code](https://github.com/nodejs/node/blob/master/lib/net.js#L1070), it looks like we only need `dns.LookupOptions`, since the caller specifies `family` and `hints`.

> I think err should be NodeJS.ErrnoException | null (should be changed at several places but this is not the point of this PR).

Makes sense.  I followed the type definitions for the `lookup(...)` functions. Would you like me to change that here, or leave it for another PR that deals with this issue globally?

> Why is family optional here?

Good catch.

I'll update the PR now...